packages/common/actions/keyStore.js

```diff
@@ -0,0 +1,16 @@
+export const ADD_TEAMS_KEY_PAIR = '@keyStore/ADD_TEAMS_KEY_PAIR';
```

Comment:
Need to add the type of a key, there is three types of a key: a personal, a team and an anonym.

Reply:
Done